charts/kserve-resources/templates/clusterservingruntimes.yaml

```diff
@@ -373,3 +373,37 @@ spec:
         limits:
           cpu: "1"
           memory: 2Gi
+---
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: deeploy-shapkernelserver
+spec:
+  annotations:
+    prometheus.kserve.io/path: /metrics
+    prometheus.kserve.io/port: "8080"
+  containers:
+  - args:
+    - --model_name={{.Name}}
+    - --predictor_host={{.Name}}-predictor.{{.Namespace}}
+    - --model_dir=/mnt/models
+    - --http_port=8080
+    - --nthread=1
+    - --explainer_sub_type=SHAPKernel
+    image: deeployml/deeploy-shap:v0.1.0
```

Comment:
Did you meant to put this as an example instead of distribution?

Reply:
I would like to contribute is as a distribution (if needed we can also open source the code). Would be nice to use it as an example for others who want to contribute a runtime.